### PR TITLE
CVSB-1262

### PR DESCRIPTION
**Pull request by mgherceanu**

### CODE REVIEW

File: src/components/signature-popover/signature-popover.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!-- Generated template for the SignaturePopoverComponent component -->
```
Comment (danieliordache) [inline src/components/signature-popover/signature-popover.html]: remove this please

File: src/components/signature-popover/signature-popover.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  confirmPop() {
+    this.loading.present();
+    this.viewCtrl.dismiss().then(
+      () => {
+        this.signatureService.saveSignature().subscribe(
+          () => {
+            this.signatureService.saveToStorage().then(() => {
+              this.loading.dismissAll();
+              this.events.publish('signature:saved');
```
Comment (danieliordache) [inline src/components/signature-popover/signature-popover.ts]: add to app.enums

File: src/components/signature-popover/signature-popover.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            });
+          },
+          () => {
+            this.loading.dismissAll();
+            this.events.publish('signature:error');
```
Comment (danieliordache) [inline src/components/signature-popover/signature-popover.ts]: add to app.enums

File: src/pages/signature-pad/signature-pad.ts [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+  ngOnInit(): void {
+    if (this.visitService.isCordova) this.screenOrientation.lock(this.screenOrientation.ORIENTATIONS.LANDSCAPE_PRIMARY);
+    this.dividerText = APP_STRINGS.SIGNATURE_DIVIDER;
+    this.underSignText = APP_STRINGS.SIGNATURE_TEXT;
+  }
+
+  ngAfterViewInit() {
+    this.signaturePad.clear();
+    // this.canvasResize();
```
Comment (danieliordache) [inline src/pages/signature-pad/signature-pad.ts]: remove comment

File: src/components/signature-popover/signature-popover.html (hunk 1 above)
Comment (mgherceanu) [reply src/components/signature-popover/signature-popover.html]: done

File: src/components/signature-popover/signature-popover.ts (hunk 3 above)
Comment (mgherceanu) [reply src/components/signature-popover/signature-popover.ts]: done

File: src/components/signature-popover/signature-popover.ts (hunk 2 above)
Comment (mgherceanu) [reply src/components/signature-popover/signature-popover.ts]: done

File: src/pages/signature-pad/signature-pad.ts (hunk 4 above)
Comment (mgherceanu) [reply src/pages/signature-pad/signature-pad.ts]: done

File: src/app/app.scss [hunk 5]
```diff
@@ -399,3 +399,23 @@ ion-item-divider.defect-heading{
...
+  }
+
+  ion-backdrop {
+    opacity: 0.4 !important;
+  }
+}
+
+.sign-toast-css {
+  .toast-message {
+    border-radius: 0.65rem !important;
```
[the remark this answers is not in the document]
Comment (mgherceanu) [reply src/app/app.scss]: in this case not, i am just reproducing an ionic feature, they use rem, better leave this one as it is

File: src/app/app.scss [hunk 6]
```diff
@@ -399,3 +399,23 @@ ion-item-divider.defect-heading{
     padding: 8px 0;
   }
 }
+
+
+//signature popover component custom css class
+.signature-popover {
+  .popover-content {
+    width: 320px;
+    background-color: #EFEEF4;
```
[the remark this answers is not in the document]
Comment (mgherceanu) [reply src/app/app.scss]: done

File: src/components/signature-popover/signature-popover.scss [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+  .row-border {
+    border-top: 0.5px groove $color-grey-3;
+
+    .col-border {
+      border-right: 0.5px groove $color-grey-3;
+    }
+
+    .button-ios {
+      height: 1.8em;
```
[the remark this answers is not in the document]
Comment (mgherceanu) [reply src/components/signature-popover/signature-popover.scss]: done

**Pull request merged**